lib/util/sdp.js

```diff
@@ -30,6 +55,21 @@ function getChromeSdpFormat() {
   return chromeSdpFormat;
 }
 
+/**
+ * Get Chrome's SDP format.
+ * @param {'plan-b'|'unified-plan'} [sdpSemantics]
+ * @param sdpSemantics
+ */
+function getChromeSdpFormat(sdpSemantics) {
+  if (!sdpSemantics || !checkIfSdpSemanticsIsSupported()) {
+    return getChromeDefaultSdpFormat();
+  }
+  return {
+    'plan-b': 'planb',
+    'unified-plan': 'unified'
+  }[sdpSemantics] || getChromeDefaultSdpFormat();
```

Comment:
if `sdpSemantics` is specified and is not a valid one, instead of silently getting `getChromeDefaultSdpFormat`, we should throw here.

Reply:
@makarandp0 Since this used only internally, I don't think we should throw here.

Reply:
I think more so reason to throw ( or not use default ) - if its internal and we are guaranteed to pass correct value - why write a fallback code. 
The disadvantage of fallback is, we would not catch a bug which caused us to  accidentally pass wrong  `sdpSemantics`